index.html

```diff
@@ -22,39 +22,58 @@
                     </a>
                     <a class="list-group-item" href="new-pipeline/evo.html">
                         Evolution Dashboard <div class="dashboard-description">View evolution of measure values over time.</div>
                     </a>
+                    <a class="list-group-item" href="https://metrics.mozilla.com/firefox-hardware-report/">
+                        Firefox Hardware Report
+                        <div class="dashboard-description">See what hardware Firefox users have.</div>
+                    </a>
                     <a class="list-group-item" href="https://sql.telemetry.mozilla.org/">
-                        SQL Portal <div class="dashboard-description">Query telemetry data with SQL.</div>
+                        SQL Portal <span class="fa fa-lock"></span>
+                        <div class="dashboard-description">View dashboards &amp; run SQL queries.</div>
+                    </a>
+                    <a class="list-group-item" href="https://moz-experiments-viewer.herokuapp.com/">
+                        Experiments viewer <span class="fa fa-lock"></span>
+                        <div class="dashboard-description">See the impact different experiments have.</div>
                     </a>
                     <a class="list-group-item" href="https://analysis.telemetry.mozilla.org/">
-                        Analysis Portal <div class="dashboard-description">Analyze telemetry data with Spark.</div>
+                        Analysis Portal <span class="fa fa-lock"></span>
+                        <div class="dashboard-description">Analyze telemetry data with Spark.</div>
                     </a>
-                    <a class="list-group-item" href="https://reports.telemetry.mozilla.org/">
-                        Reports Portal <div class="dashboard-description">Explore data analyses &amp; reports.</div>
+                </div>
+                <h3>Documentation</h3>
+                <div class="dashboard-list list-group" style="margin: 0">
+                    <a class="list-group-item" href="https://docs.telemetry.mozilla.org/">
+                        Data docs <div class="dashboard-description">Find what data is available &amp; how to use it.</div>
+                    </a>
+                    <a class="list-group-item" href="https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/">
+                        Firefox Telemetry docs <div class="dashboard-description">Learn how to record data &amp; what data gets sent.</div>
+                    </a>
+                    <a class="list-group-item" href="https://georgf.github.io/fx-data-explorer/">
+                        Probe dictionary <div class="dashboard-description">Find what probes are submitted from Firefox.</div>
+                    </a>
+                    <a class="list-group-item" href="https://wiki.mozilla.org/Firefox/Shield">
```

Comment:
Kudos that (as before) the entire rectangles of the list-group-items are clickable.